### PR TITLE
Faster Clustering: Doe Downsampling work?

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
So I did a few experiments with downsampling on some of the larger datasets. We seem to get quite consistent performance, but seems like it is consistently slightly higher scores for smaller samples, though the std seems to remain similar. 

This seems to indicate that we can't just keep the existing scores, but estimate them using lower sample size.

### CODE REVIEW

File: mteb/tasks/BitextMining/multilingual/BUCCBitextMining.py
Comment (KennethEnevoldsen) [inline mteb/tasks/BitextMining/multilingual/BUCCBitextMining.py:1]: feel free to ignore this

File: mteb/abstasks/AbsTaskClustering.py [hunk 1]
```diff
@@ -66,4 +66,4 @@ def _evaluate_monolingual(self, model, dataset, split="test", **kwargs):
 
         v_mean = np.mean(v_measures)
         v_std = np.std(v_measures)
-        return {"v_measure": v_mean, "v_measure_std": v_std}
+        return {"v_measure": v_mean, "v_measure_std": v_std, "v_measures": v_measures}
```
Comment (KennethEnevoldsen) [inline mteb/abstasks/AbsTaskClustering.py:69]: to allow user to calculate ci etc.

File: tmp/slowest_segment.py
Comment (KennethEnevoldsen) [inline tmp/slowest_segment.py:1]: feel free to ignore

**Comment by Muennighoff** [comment]
Agree with what has been said - we should make sure we're not losing precision in the task & update all scores if they change👍

**Comment by KennethEnevoldsen** [comment]
superseded by #481

**Pull request closed without merging**